src/app/Component/RegisterUser/RegisterUser.js

```diff
@@ -194,10 +194,10 @@ class Registration extends Component {
             //             {this.errors.password}
             //         </div>
             //         <div className='form-group'>
-            //                 <input className='form-control' type='password' name='cpassword' placeholder="Confirm Password" ref={this.confirmpassword} onBlur={this.validationForm.bind(this, 'password')} ></input>
+            //                 <input className='form-control' type='password' name='confirmpassword' placeholder="Confirm Password" ref={this.confirmpassword} onBlur={this.validationForm.bind(this, 'password')} ></input>
```

Comment:
this is commented right why you are changing again here .Remove commented lines if it is not used